CONTRIBUTORS

```diff
@@ -9,3 +9,4 @@ If you think you should be in this list and don't find yourself, write to floren
 and we'll fix it!
 
 Contributors:
+Copyright (c) 2020 Antmicro <www.antmicro.com>
```

Comment:
No `Copyright (c) 2020` here. 

Probably should add myself and Florent here too.

Reply:
fixed